ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -266,3 +266,9 @@ def linspace(start, stop, num, name=None, axis=0):
 @to_ivy_arrays_and_back
 def realdiv(x, y, name=None):
     return ivy.divide(x, y)
+
+
+@to_ivy_arrays_and_back
+def tile(input, multiples, name=None):
+    return ivy.einops_repeat(
+        input, 'h w -> (h1 h) (w1 w)', h1=multiples[0], w1=multiples[1])
```

Comment:
Why did you use `einops_repeat` in the first place? I just realised that is the problem after all😅

Reply:
I think you should use `ivy.tile`, where `repeats` would be `multiples`, and we only need one array, not two.

Reply:
> Why did you use einops_repeat in the first place? I just realised that is the problem after all😅

at the time, I couldn't find an equivalent function and wanted to circumvent creating a PR for a new one 😅

Reply:
[is this the function you are referring to?](https://lets-unify.ai/ivy/functional/ivy/manipulation/tile/tile_functional.html)

Reply:
okay, according to the documentation, I will edit my function code like so:

```python
@to_ivy_arrays_and_back
def tile(input, multiples, name=None):
    return ivy.tile(input, multiples)
```